index.html

```diff
@@ -117,6 +117,34 @@ <h3>Concept and Initial Report</h3>
       <option value="Transit Oriented">Transit Oriented</option>
       <option value="Main Street/Special">Main Street/Special</option>
     </select>
+    <div class="population-slider">
+      <div class="slider-label">
+        <span class="lower value"></span>
+        <span> - </span>
+        <span class="upper value"></span>
+      </div>
+      <div class="min-max-slider" data-legendnum="12">
+        <input
+          id="left-slider"
+          class="left-slider"
+          name="min"
+          type="range"
+          step="0.5"
+          min="0"
+          max="11"
```

Comment:
How do these get determined? If we had fewer options per https://github.com/ParkingReformNetwork/reform-map/issues/231, would these change?